### PR TITLE
Add hooks example and update named templates

### PR DESCRIPTION
**Pull request by hairmare**
This contains some minor cleanup, mostly in the boilerplate parts. It also adds a largish section on hooks and some infos on helm starters in the advanced parts.

The actual content was cobbled together pretty quickly but getting everything to render properly was quite a fight to say the least.

### CODE REVIEW

File: kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 % Helm
 % Lucas Bickel
 
+![](static/adfinis_sygroup_logo.png)
+
+Be smart. Think open source.
+
 # ![](static/helm_logo_transparent.png)
```
Comment (eyenx) [inline kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md]:
```suggestion
```

File: kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md [hunk 2]
```diff
@@ -277,12 +282,12 @@ helm get manifest my-release
 ## Named Templates
 
 * `define` declares a new named template inside of your template
-* `template` imports a named template
+* `include` used a named template
```
Comment (eyenx) [inline kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md]:
```suggestion
* `include` uses a named template
```

File: kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md [hunk 3]
```diff
@@ -321,7 +328,65 @@ metadata:
...
+## Delete Hooks
+
+* pre-delete
+* post-delete
+
+## CRD Hooks
+
+* crd-install
+
+Is used on Custom Resource Definitions to ensure that they are
```
Comment (eyenx) [inline kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md]: I had to read this three times to understand it right. Perhaps it's only me, but I would prefer a simpler sentence here.

File: kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md (hunk 1 above)
Comment (hairmare) [reply kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md]: Why? The logo should be in the training somewhere. It serves as an important visual identifier.

File: kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md (hunk 2 above)
Comment (hairmare) [reply kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md]: Fixed

File: kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md (hunk 3 above)
Comment (hairmare) [reply kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md]: Change to the following which I'm assuming has a better readability:

> Used on Custom Resource Definitions to ensure that they are
> defined before they are used by other manifests in the chart.

File: kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md (hunk 1 above)
Comment (eyenx) [reply kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md]: but its commented out?

Comment (hairmare) [reply kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md]: It's not a comment, it's a h1 markdown tag that forces the logo to be on it's own page.

Comment (eyenx) [reply kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md]: oops

**Comment by eyenx** [comment]
LGTM

**Pull request merged**